js/lykke2.js

```diff
@@ -0,0 +1,1171 @@
+        //             }
+        //         ],
+        //         "error":null
+        //     }
+        //
+        const orderbook = this.safeValue (payload, 0, {});
+        const timestamp = this.safeString (orderbook, 'timestamp');
+        return this.parseOrderBook (orderbook, symbol, timestamp, 'bids', 'asks', 'p', 'v');
+    }
+
+    parseTrade (trade, market) {
+        //
+        //  public fetchTrades
+        //
+        //     {
+        //         "id":"71df1f0c-be4e-4d45-b809-c108fad5f2a8",
+        //         "assetPairId":"BTCUSD",
+        //         "timestamp":1643345958414,
+        //         "volume":0.00010996,
+        //         "price":37205.723,
+        //         "side":"buy"
+        //      }
+        //
+        //  private fetchMyTrades
+        //         {
+        //             "id":"813a3ffa-1c4b-45cb-b13f-1c077ea2748b",
+        //             "timestamp":1644155923357,
+        //             "assetPairId":"BCHEUR",
+        //             "orderId":"1b367978-7e4f-454b-b870-64040d484443",
+        //             "role":"Taker",
+        //             "side":"sell",
+        //             "price":280.569,
+        //             "baseVolume":0.01,
+        //             "quoteVolume":2.8056,
+        //             "baseAssetId":"2a34d6a6-5839-40e5-836f-c1178fa09b89",
+        //             "quoteAssetId":"EUR",
+        //             "fee":null
+        //         }
+        //
+        const marketId = this.safeString (trade, 'assetPairId');
+        market = this.safeMarket (marketId, market);
+        const symbol = market['symbol'];
+        const id = this.safeString2 (trade, 'id', 'id');
+        const orderId = this.safeString (trade, 'orderId');
+        const timestamp = this.safeInteger (trade, 'timestamp');
+        const price = this.safeString2 (trade, 'price', 'price');
+        let amount = this.safeString2 (trade, 'volume', 'amount');
+        if (amount === undefined) {
+            amount = this.safeString2 (trade, 'baseVolume', 'amount');
+        }
+        const side = this.safeStringLower (trade, 'side');
+        const fee = {
+            'cost': this.parseNumber ('0'), // There are no fees for trading.
+            'currency': market['quote'],
+        };
+        return this.safeTrade ({
+            'id': id,
+            'info': trade,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'symbol': symbol,
+            'type': undefined,
+            'order': orderId,
+            'side': side,
+            'takerOrMaker': undefined,
+            'price': price,
+            'amount': amount,
+            'cost': undefined,
+            'fee': fee,
+        }, market);
+    }
+
+    async fetchTrades (symbol, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        if (limit === undefined) {
+            limit = 100;
```

Comment:
do we need to define a limit here?

the lykke api documentation for `take` is the following - "(optional) Take the specified number of elements." therefore we should not send it unless the user specifies it